File: Data Structures & Algorithms/counting-bits/submission-0.cpp
class Solution {
public:
    int hammingWeight(uint32_t n) {
        int count = 0;
        for (int i = 0; i < 32; i++) {
            count += (1 & n);
            n >>= 1;
        }

        return count;
    }
    vector<int> countBits(int n) {
        vector<int> result;

        for (int i = 0; i <= n; i++) {
            result.push_back(hammingWeight(i));
        } 

        return result;
        
    }
};

File: Data Structures & Algorithms/majority-element/submission-0.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        count = 0
        n = None
        m = {}
        for num in nums:
            if num in m:
                m[num] += 1
            else:
                m[num] = 1

            if m[num] > count:
                count = m[num]
                n = num

        return n
        